src/lambda/resourceRouter.ts

```diff
@@ -1,32 +1,33 @@
-import {notFound, badRequest} from '../libs/buildResponse';
-import httpMethods from '../libs/httpMethods';
+import {notFound, badRequest} from '../http/buildResponse';
+import {HTTPMethod} from '../http/httpMethods';
 import normalizeUrl from 'normalize-url';
+import {ApiGatewayEvent, ResourceFunction, RouteResource} from './models';
 
 const beginningSlashes = /^\/+/;
 const separatorRegex = /\/+/;
 const tokenRegex = /%7B\w+%7D/;
 
-const testToken = (str) => {
+const testToken = (str: string) => {
   return str && str.match(tokenRegex);
 };
 
-const canonicalizePath = (path) => {
+const canonicalizePath = (path: string) => {
   return normalizeUrl(
     path.replace(beginningSlashes, ''),
     {
       stripProtocol: true,
       stripHash: true,
-      removeQueryParameters: true,
```

Comment:
just question -- why are we keeping in query params here?

Reply:
removeQueryParameters doesn't take in a boolean, it takes in a string regex. This line wasn't doing anything, so I took it out.